worker/transcribee_worker/identify_speakers.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Callable
+
+import numpy as np
+import numpy.typing as npt
+import torch
+from sklearn.cluster import AgglomerativeClustering
+from speechbrain.pretrained import EncoderClassifier
+from transcribee_proto.document import Document
+
+from .config import settings
+
+
+def identify_speakers(
+    audio: npt.NDArray, doc: Document, progress_callback: Callable[[str, float], None]
+):
+    logging.info("Running Speaker Identification")
+
```

Comment:
This crashes if <2 paragraphs in 1 document

```suggestion
    if len(doc.children) == 0:
        return
    elif len(doc.children) == 1:
        doc.children[0].speaker = 1
        return
```